grid-client/deployer/node_filter.go

```diff
@@ -246,11 +247,17 @@ func serializeOptions(options types.NodeFilter) (string, error) {
 	if err != nil {
 		return "", nil
 	}
-	filterString, err := json.Marshal(params)
-	if err != nil {
-		return "", nil
+
+	var filterStringBuilder strings.Builder
```

Comment:
please add example on what is expected here, is that ur.Values{} that you're encoding or something else?

also change names filterStringBuilder to `sb` and filterString to filter or qs